ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -3,6 +3,10 @@
 
 @implementation CodePushUpdateUtils
 
+NSString * const ManifestFolderPrefix = @"CodePush";
+NSString * const DefaultJsBundleName = @"main.jsbundle";
```

Comment:
This seems a little brittle since we know people can rename it. Shouldn't we try to get the actual bundle file name from the CodePush class?